src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php

```diff
@@ -76,6 +76,9 @@ public function __construct(
 
     public function warmUp()
     {
+        if ($this->cache->isFresh()) {
```

Comment:
This reads very strange... If the cache is still fresh, there is actually no point of invalidating it, isn't it? But I have seen that the `isFresh` call just checks for the existence of the file... Since it reads so strange I would either simply always call `$this->cache->invalidate()`. If it causes any problems when the file is not existing, I would handle that in the `invalidate` method (just do nothing then). I think that would be a reasonable behavior, since the end result would be the same.